projects/marketing-site/components/Proposals/ProposalL2.tsx

```diff
@@ -11,7 +11,7 @@ import Link from "next/link";
 
 export default function ProposalL2() {
   return (
-    <div className="relative overflow-hidden rounded-lg bg-white py-16">
+    <div className="relative overflow-hidden rounded-lg dark:bg-white bg-slate-100 py-16">
```

Comment:
i think the slate color is a little too dark for the little circles on the page. could you just make it bg-white always?